src/apis/message/queries.ts

```diff
@@ -0,0 +1,40 @@
+import { useMutation, useQuery } from '@tanstack/react-query'
+import {
+  getMessageRooms,
+  getMessage,
+  createMessage,
+  deleteMessageRoom
+} from './apis'
+import type {
+  CreateMessageReq,
+  GetMessageReq,
+  GetMessageRoomsReq
+} from './types'
+
+export const useGetMessageRooms = (params: GetMessageRoomsReq) =>
+  useQuery({
+    queryKey: ['getMessageRooms'],
+    queryFn: () => getMessageRooms(params)
+  })
+
+export const useGetMessageQuery = ({ msgRoomId, ...res }: GetMessageReq) =>
```

Comment:
```suggestion
export const useGetMessageQuery = ({ msgRoomId, ...params }: GetMessageReq) =>
```
네이밍 수정되면 좋을 것 같습니다!
여기두 위처럼 params 변경되었을 때, refetch되지 않아도 되는지 같이 확인 부탁드립니다!

Reply:
[반영 완료 ](https://github.com/price-offer/offer-fe/pull/196/commits/cd910468f471f328bef8a5281af6182ca20cec16#diff-12a235e447906f2933ad63c33beb028650446faf70785737178bc5ed191ef693R18)!